[elasticsearch] Helm lint fails on chart v7.15.0

**Chart version:** 7.15.0

**Helm Version:** 3.7.1

**Operating System:** MacOS

**Describe the bug:**

**Steps to reproduce:**

1. Create new chart with elasticsearch as sub-chart

```yaml
dependencies:
  # Official chart from : https://github.com/elastic/helm-charts/tree/master/elasticsearch
  - name: elasticsearch
    version: 7.15.x
    repository: "https://helm.elastic.co"
    condition: elasticsearch.enabled, global.elasticsearch.enabled
```

2. `helm dep up <newchart>`
3. `helm lint --with-subcharts <newchart>` 

```
❯ helm lint --with-subcharts charts/newchart
==> Linting charts/newchart/charts/elasticsearch-7.15.0.tgz
[ERROR] templates/service.yaml: unable to parse YAML: invalid Yaml document separator: kind: Service
[ERROR] templates/test/test-elasticsearch-health.yaml: unable to parse YAML: invalid Yaml document separator: apiVersion: v1
```

**Expected behavior:**

Lint will pass.

**Any additional context:**

Changing the chart version to `7.14.0` results in no linting errors.

```
❯ helm lint --with-subcharts charts/newchart
==> Linting charts/newchart/charts/elasticsearch-7.14.0.tgz
```

--- a/elasticsearch/templates/poddisruptionbudget.yaml
+++ b/elasticsearch/templates/poddisruptionbudget.yaml
@@ -1,5 +1,5 @@
----
 {{- if .Values.maxUnavailable }}
+---
 apiVersion: policy/v1beta1
 kind: PodDisruptionBudget
 metadata:

--- a/elasticsearch/templates/service.yaml
+++ b/elasticsearch/templates/service.yaml
@@ -1,5 +1,5 @@
----
 {{- if .Values.service.enabled -}}
+---
 kind: Service
 apiVersion: v1
 metadata:

--- a/elasticsearch/templates/test/test-elasticsearch-health.yaml
+++ b/elasticsearch/templates/test/test-elasticsearch-health.yaml
@@ -1,5 +1,5 @@
----
 {{- if .Values.tests.enabled -}}
+---
 apiVersion: v1
 kind: Pod
 metadata:

